launcher/ui/GameListSelector.py

```diff
@@ -35,7 +34,7 @@ def on_setting(self, key, value):
                     if item[0] == old_list_uuid:
                         if self.get_index() != i:
                             print("- set choice index to", i)
-                            self.set_index(i, signal=False)
+                            self.set_index(i)
```

Comment:
`signal` is already False by default in your method override, so no need to explicitely set it